### DISCUSSION TITLE
Export title page/manuscript seperately for academic double-blind submissions.

### DISCUSSION THREAD

**Discussion by blenback** [quarto-dev/quarto-cli#6743] [Feature Requests]
### Description

In the scientific disciplines I write publications in it is common for journals that use double-blind reviewing to request seperate document uploads of:
- the manuscript without author information
- A title page with author information, affiliations etc. 

I know I can produce the former with the `formatting: doubleblind` argument in the YAML and I could do the latter with a seperate .qmd file of only the author information but it would be great if there was an option to simultaneously save a copy of the latter i.e. a pdf/html output of only the title and author details.

**Comment by cscheid** [comment]
I used to be a prof in a discipline with double-blind reviewing as well! The way this works (or worked!) at the time is that the differences between the submission and the final version of the paper were dictated by the template from the journal or conference. As a result, it wasn't something that could be done generically. Not coincidentally, this is how quarto currently handles this: some of our [journal templates](https://quarto.org/docs/extensions/listing-journals.html) do support anonymous submissions.

In your field, do you typically submit a manuscript in a generic format or do you need to use a template? I can see this making sense if it's the former. In which case, if there's enough other folks in the same situation, then we could try to do something to our default PDF template so it respects anonymity requirements.

**Reply by mcanouil** [reply]
FYI, I believe a filter could do the job, such as the one proposed in https://github.com/quarto-dev/quarto-cli/discussions/3005#discussioncomment-6926776

**Reply by blenback** [reply]
In my field a lot of journals under certain publishers call their submission system 'your paper your way' which doesn't require so much strict formatting, although they often do make specific requests, but ultimately no strict template. However, the main annoyance, in the case of double-blind submissions is still that I need to submit one seperate title page doc. and one seperate manuscript doc. with the title but without any author details. I was hoping for a smooth way to automate the creation of these, when rendering the .qmd.

**Reply by mcanouil** [reply]
I realise that conditional content and project profiles might be of help here:
- https://quarto.org/docs/authoring/conditional.html
- https://quarto.org/docs/projects/profiles.html

**Reply by dragonstyle** [reply]
For the JASA template, I added support for an option to control the output of author information (taken really from their template which supports this). It is controlled by yaml using the option `blinded` (set it true and author information isn't included in the document). I was thinking users would then do something like this in their document to target review vs real outputs:

```yaml
---
title: My Article
format:
  jasa-pdf:
    output-file: article.pdf
  jasa-pdf+review:
    blinded: true
    output-file: article-review.pdf
---

...

```

The user could then use render commands to control which output they'd like:

```bash
quarto render --to jasa-pdf
quarto render --to jasa-pdf+review
```

HTH!

**Comment by tpoisot** [comment]
I've been (slowly) working on a generic preprint template to use in the lab, and I built this in with document options: https://github.com/tpoisot/preprint-template

there's one for the title page (make it anonymous), and one to generate an author fact sheet at the end with complete affiliations

It's a *huge* ~work in progress~ mess, but we used it for a submission just last week
